src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = sourceString
+    .split(';')
+    .map((itemFromArr) => itemFromArr.trim())
+    .filter((itemFromArr) => itemFromArr.length > 0)
+    .map((itemFromArr) => {
+      const separatorIndex = itemFromArr.indexOf(':');
+      const newKey = itemFromArr.slice(0, separatorIndex).trim();
+      const newValue = itemFromArr.slice(separatorIndex + 1).trim();
+
+      return [newKey, newValue];
+    })
+    .reduce((acc, [key, value]) => {
+      acc[key] = value;
+
+      return acc;
```

Comment:
The accumulator parameter 'acc' in the 'reduce' function could be given a more descriptive name such as 'styleObject' to better represent the resulting object of CSS styles.